### PR TITLE
[PR-9901] Create Events doc

### PR DESCRIPTION
**Pull request by iurevych**
## Demo
http://void-docs.talkable.com/advanced_features/events.html

## Related Stories
[](https://talkable.atlassian.net/browse/PR-9901)

### CODE REVIEW

**Comment by iurevych** [comment]
@vitalikdanchenko would be great if you can read through the doc as well.

**Comment by Christian-Wathne** [comment]
Hi @iurevych ; I wasn't sure how to edit your code directly. The first few pages of this google doc are my edits.... https://docs.google.com/document/d/1K5bsHhizF1tkj7VZL4O20cmrCRzeUN8JbG2HO0lD96A/edit?usp=sharing

**Comment by iurevych** [comment]
@zhuravel ready for review.

File: source/advanced_features/events.rst [hunk 1]
```diff
@@ -22,7 +22,7 @@ whether the event was associated with a referral or not. With both “subscripti
 to report on and reward these events as desired. See some examples below.
 
 Subscription payments should go under “subscription” event category. All such events
-can be found inside Reports |rarr| Events. Here is an example:
+can be found inside Reports → Events. Here is an example:
```
Comment (zhuravel) [inline source/advanced_features/events.rst]: Okay, then let’s replace `|rarr|` everywhere across the docs.

**Pull request merged**